app/controllers/facility_account_users_controller.rb

```diff
@@ -33,7 +34,6 @@ def new
 
   # POST /facilities/:facility_id/accounts/:account_id/account_users
   def create
-    @user = User.find(params[:user_id])
     @account_user = AccountUser.grant(@user, create_params[:user_role], @account, by: session_user)
 
     if @account_user.persisted?
```

Comment:
What do you think about adding a branch to the existing conditional like:
```suggestion
    if @account_user.nil?
      flash[:error] = text("create.existing_member_error", user: @user.full_name)
      redirect_to facility_account_members_path(current_facility, @account)
    elsif @account_user.persisted?
```
This logic isn't as specific but I find the expected outcomes easier to parse.  

You could also do something like:
```'
def existing_member_becoming_owner?
  create_params[:user_role] == AccountUser::ACCOUNT_OWNER &&
    @account.account_users.map(&:user_id).include?(@user.id)
end
```
to cover this specific case, but I'm ok with using the more specific error message to cover any nil account_user.
We can adjust in the future if we need to.